Mathlib/Data/Finsupp/Order.lean

```diff
@@ -0,0 +1,268 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Aaron Anderson
+
+! This file was ported from Lean 3 source module data.finsupp.order
+! leanprover-community/mathlib commit f7fc89d5d5ff1db2d1242c7bb0e9062ce47ef47c
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.Finsupp.Defs
+
+/-!
+# Pointwise order on finitely supported functions
+
+This file lifts order structures on `α` to `ι →₀ α`.
+
+## Main declarations
+
+* `Finsupp.orderEmbeddingToFun`: The order embedding from finitely supported functions to
+  functions.
+-/
+
+-- porting notes: pulled this out of module documentation becasue it doesn't exist?
```

Comment:
It probably moved to a different file in mathlib3, and the module docs weren't updated.
Voila: https://leanprover-community.github.io/mathlib_docs/data/finsupp/multiset.html#finsupp.order_iso_multiset
So please turn the porting note into a TODO